test/tests/neml_stochasticSimple/le_stress_simple.i

```diff
@@ -0,0 +1,287 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+[]
+
+[Kernels]
+  [TensorMechanics]
+  []
+[]
+
+[AuxVariables]
+  [strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_yz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_xz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_yz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[AuxKernels]
```

Comment:
We can get rid of all of this lengthy input syntax by using the `TensorMechanics/Master` action with `generate_outputs`.  Doing it this way isn't technically wrong, but maybe as a follow-on, you could clean this up. We have a ton of inputs set up the old way that we need to convert.

Reply:
I was going to remove all of the [./var] and [../] from the input files in the blackbear tests so I could do this too.  I'll open a separate issue for that.  Right now there isn't a general issue for clean-up.

Reply:
If you want to remove the `./` stuff, I'd highly recommend using `hit`.  It's in the moose contrib somewhere, I think.  You can just do something like `hit format file.i`, and it will indent and clean up the file. I'd like to figure out how to start running that in our pre-tests just like we run clang-format so we never have to worry about it again.

Reply:
Feel free to make an issue for general test cleanup, or just make an issue for these specific things. Converting tests to use TM master is a little more involved.